practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/model/Practitioner.java

```diff
@@ -0,0 +1,22 @@
+package uk.ac.ox.ndph.mts.practitioner_service.model;
+
+import java.io.Serializable;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.RequiredArgsConstructor;
+import lombok.NonNull;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class Practitioner implements Serializable {
+
+    private String prefix;
+    private String givenName;
+    private String familyName;
```

Comment:
I am not sure , but wanted to check anyway, if I look at Practitioner name which is a HumanName FHIR datatype (https://www.hl7.org/fhir/datatypes.html#HumanName) then givenName and prefix are arrays, am I right?